contracts/RewardsPoolFactory.sol

```diff
@@ -54,21 +77,37 @@ contract RewardsPoolFactory is Ownable {
         address rewardsPoolAddress,
         address extendRewardToken,
         uint256 extendRewardAmount
-    )
-        external
-        onlyOwner
-    {
-        require(extendRewardAmount != 0, 'RewardsPoolFactory::extendRewardPeriod: Reward must be greater than zero');
-
-        require(rewardsPoolAddress != address(0), 'RewardsPoolFactory::extendRewardPeriod: not deployed');
-        require(hasStakingStarted(rewardsPoolAddress), 'RewardsPoolFactory::extendRewardPeriod: Staking has not started');
-
-        (uint256 rate, , , ,) = RewardsPool(rewardsPoolAddress).rewardsTokensMap(extendRewardToken);
-
-        require(rate != 0, 'RewardsPoolFactory::extendRewardPeriod: Token for extending reward is not known'); // its expected that valid token should have a valid rate
-
-        IERC20Detailed(extendRewardToken).safeApprove(rewardsPoolAddress, extendRewardAmount);
-        RewardsPool(rewardsPoolAddress).addRewards(extendRewardToken, extendRewardAmount);
+    ) external onlyOwner {
+        require(
+            extendRewardAmount != 0,
+            "RewardsPoolFactory::extendRewardPeriod: Reward must be greater than zero"
+        );
+
+        require(
+            rewardsPoolAddress != address(0),
+            "RewardsPoolFactory::extendRewardPeriod: not deployed"
+        );
+        require(
+            hasStakingStarted(rewardsPoolAddress),
+            "RewardsPoolFactory::extendRewardPeriod: Staking has not started"
+        );
+
+        (uint256 rate, , , , ) =
+            RewardsPool(rewardsPoolAddress).rewardsTokensMap(extendRewardToken);
+
+        require(
+            rate != 0,
+            "RewardsPoolFactory::extendRewardPeriod: Token for extending reward is not known"
+        ); // its expected that valid token should have a valid rate
+
+        IERC20Detailed(extendRewardToken).safeApprove(
+            rewardsPoolAddress,
+            extendRewardAmount
+        );
+        RewardsPool(rewardsPoolAddress).addRewards(
```

Comment:
I think the function is called extend. Also we should be doing this with `transfer` instead of approve and transferFrom